wiki/Game_modifier/Auto/en.md

```diff
@@ -36,11 +36,11 @@ The mod functions exactly like a [replay](/wiki/Replay), with the options for 0.
 
 ### osu!standard
 
-In [osu!standard](/wiki/Game_mode/osu!), when encountered with a spinner, the Auto mod will spin to a maximum of 477 spins per minute (SPM) in a counter-clockwise rotation.
```

Comment:
i don't understand this change. seemed fine?

Reply:
Anticlockwise is the British term